flight/PiOS/STM32/pios_irq.c

```diff
@@ -49,6 +47,7 @@ static uint32_t prev_primask;
 */
 int32_t PIOS_IRQ_Disable(void)
 {
+	/* XXX TODO: This is racy */
```

Comment:
It requires an atomic / analysis (because there could be an intervening task switch).  I just spotted it while I was here, but thinking of the correct fix takes some effort.